HousingRepairsOnlineApi/Helpers/IRepairBookingResponseHelper.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Collections.Generic;
+using HousingRepairsOnlineApi.Domain;
+using Microsoft.Azure.Cosmos;
+
+namespace HousingRepairsOnlineApi.Helpers
+{
+    public interface IRepairBookingResponseHelper
+    {
+        dynamic GetRepairBookingResponse(Repair repair);
```

Comment:
Following the above comment ([here](https://github.com/Newarkandsherwood/housing-repairs-online-api/pull/43/files#diff-925127bc52a7bb013fa8d7a9290a2a15726c88e6ace60b05676b2398d298b253R7)), this could be renamed to `Map` too.